<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomatonView</class>
 <widget class="QWidget" name="AutomatonView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Automaton:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mAutomatonCombo">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="mPreprocessingCheckBox">
       <property name="text">
        <string>Preprocessing</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="mGraphicsView">
     <property name="backgroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mAutomatonCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AutomatonView</receiver>
   <slot>selectAutomaton(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mPreprocessingCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>AutomatonView</receiver>
   <slot>enablePreprocessing(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectAutomaton(int)</slot>
  <slot>enablePreprocessing(bool)</slot>
 </slots>
</ui>
